SparseGrids/Benchmarks/benchCommon.hpp

```diff
@@ -9,14 +9,16 @@ enum BenchFuction{
     bench_none,
     bench_make,
     bench_loadneeded,
-    bench_evaluate
+    bench_evaluate,
+    bench_weights
```

Comment:
* `weights` is too generic since there are many different types of weights, use `iweights`